addons/crm/models/res_partner.py

```diff
@@ -35,14 +35,18 @@ def default_get(self, fields):
                 )
         return rec
 
-    def _compute_opportunity_count(self):
+    def _get_all_partners(self):
         # retrieve all children partners and prefetch 'parent_id' on them
-        all_partners = self.with_context(active_test=False).search_fetch(
+        return self.with_context(active_test=False).search_fetch(
             [('id', 'child_of', self.ids)], ['parent_id'],
-        )
+        ).ids
 
+    def _get_opportunity_domian(self):
```

Comment:
Beware typos ! (and naming maybe ?)
```suggestion
    def _get_contact_opportunities_domain(self):
```